recipes/recipes_emscripten/lpython/recipe.yaml

```diff
@@ -1,48 +0,0 @@
-context:
-  name: lpython
-  version: "0.21.2"
-
-package:
-  name: "{{ name }}"
-  version: '{{ version }}'
-
-source:
-  url: https://github.com/lcompilers/lpython/releases/download/v{{ version }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 7ec55e398c4007f49af0e44b3599ae3be39e831f6d7696e2b65e80514a8d15b6
-  patches:
-    - patches/remove-static-assert.patch
-
-build:
-  number: 0
-
-requirements:
-  build:
-    - "{{ compiler('c') }}"
-    - "{{ compiler('cxx') }}"
-    - cmake
-    - make   # [unix]
-    - zlib=1.2.13
-    # - xeus =3.0.5       # [build_platform != target_platform]
-    # - xeus-zmq =1.0.2   # [build_platform != target_platform]
-    # - llvmdev =16.0.6   # [build_platform != target_platform]
-    # - xtl               # [build_platform != target_platform]
-    # - nlohmann_json =3.11.2     # [build_platform != target_platform]
-    # - cppzmq            # [build_platform != target_platform]
-    # - zlib              # [build_platform != target_platform]
-    # - zstd-static =1.5.5  # [build_platform != target_platform]
-  host:
-    - zlib
-
-about:
-  home: https://lpython.org
-  license: BSD-3-Clause
-  license_family: BSD
-  license_file: LICENSE
-  summary: Modern interactive LLVM-based Python compiler
-  dev_url: https://github.com/lcompilers/lpython.git
-
-extra:
-  recipe-maintainers:
-    - certik
-    - anutosh491
-    - Shaikh-Ubaid
```

Comment:
Wanted to look at lpython's recipe for something but realized the file was deleted :(
Is there a reason this was done @DerThorsten ?

If not I could raise a PR fixing this cause that only leaves us with the `build.sh` file and the `patches` dir in the lpython folder.

Reply:
No, that was not on purpose, feel free to re-add it.

Sorry :/

Reply:
No issues :)